src/explainer_prep.R

```diff
@@ -0,0 +1,346 @@
+    ) +
+    theme_flowfacet(base = 12, color_bknd, text_color) +
+    theme(axis.text.y = 
+            element_text(size = axis_text_size,
+                         vjust = 0.5, #c(1, 0), 
+                         hjust = 1),
+          axis.title.y = element_text(size = axis_title_bottom_size,
+                                      margin = margin(r = 5)),
+          axis.title.x.bottom = element_text(size = axis_title_bottom_size,
+                                             vjust = -1,
+                                             margin = margin(t = 5)),
+          axis.title.x.top = element_text(size = axis_title_top_size,
+                                          vjust = 0,
+                                          margin = margin(b = -5)),
+          axis.text.x.bottom = element_text(size = axis_text_size,
+                                            vjust = 1,
+                                            # nudge labels up closer to bottom
+                                            margin = margin(t = -7))) +
+    scale_x_date(breaks = seq.Date(date_start, date_end, "1 week"),
+                 position = "bottom",
+                 labels = lubridate::day(seq.Date(date_start, date_end, "1 week")),
+                 sec.axis = dup_axis(
+                   name = "National"
+                 )) +
+    coord_fixed(ratio = 28, clip = "off")
+  
+}
+
+#' @description Extract and restyle legend
+#' @param plot_nat National data plot
+#' @param barwidth Numeric: Define width of each bar in percentile bin legend
+#' @param barheight Numeric: Define height of each bar in percentile bin legend
+#' @param text_size Numeric: Define size of legend text
+restyle_legend_explainer <- function(plot_nat, barwidth, barheight, text_size){
+  # Restyle legend 
+  plot_nat_legend <- plot_nat +
+    guides(fill = guide_colorsteps(
+      title = "",
+      nrow = 1,
+      direction = 'horizontal',
+      label.position = "bottom",
+      barwidth = barwidth,
+      barheight = barheight,
+      background = element_rect(fill = NA),
+      show.limits = TRUE
+    )) +
+    theme(legend.background = element_rect(fill = NA),
+          text = element_text(family = font_legend, color = text_color, size = text_size))
+  
+  legend <- get_legend(plot_nat_legend)
+  
+}
+
+#' @description Cowplot national data image with explainer annotations
+#' @param plot_nat National data plot
+#' @param date_start first day of focal month
+#' @param flow_label legend explainer text
+#' @param source_label data source text
+#' @param legend restyled legend
+#' @param explainer_label top annotation for how to read the plot
+#' @param file_png file path for final png
+#' @param width width of final png
+#' @param height height of final png
+#' @param font_legend font used for legend text
+#' @param text_color color used for viz text
+cowplot_national_explainer <- function(plot_nat, date_start, flow_label, 
```

Comment:
Instead of duplicating the plotting function and adding to it - use the existing plotting function for this chart and add annotations to it. That way if we ever change any of the plot settings or layout we will only need to change it in one place, reducing error and effort needed.

There are miultiple ways this could be done:
1.  by exporting as a png, reading back in, and adding the text and arrow elements.
2. generating the cowplot in one target, and using separate targets for the export steps (one being the original appearance, another being the one with these added annotations)